presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -525,7 +526,12 @@ public SystemSessionProperties(
                         null,
                         false,
                         value -> min(taskManagerConfig.getMaxDriversPerTask(), validateNullablePositiveIntegerValue(value, MAX_DRIVERS_PER_TASK)),
-                        object -> object));
+                        object -> object),
+                booleanProperty(
+                        IGNORE_STATS_CALCULATOR_FAILURES,
+                        "Experimental: Ignore statistics calculator failures rather than failing",
```

Comment:
I don't think it should be marked experimental.  It's a safety mechanism.

Reply:
+1